src/main.ts

```diff
@@ -0,0 +1,4 @@
+import * as Synthetics from './commands/synthetics/main'
+import {parseConfigFile} from './helpers/utils'
+
+export {parseConfigFile, Synthetics}
```

Comment:
Not sure about that design though.

I don't see it scaling very well with other commands.

Also, why the uppercase on `Synthetics`? It's not a class.

I believe this needs to be discussed a bit more and prepared for the future as well.
This will become the main entrypoint of datadog-ci as a lib, it needs to be thought out. cc @m-rousse 

IMO, we should simplify this, having a `index.ts` in each command directory that will handle what's getting exported to the lib (and rename the current `index.ts` as `cli.ts` for the automatic cli registration).

Then, in the root `package.json` we keep the `"main"` to be `"dist/index.ts"`.
Renaming the current `index.ts` to `cli.ts` and update the `"bin"`, as suggested by @etnbrd in another comment.

Reply:
> Also, why the uppercase on `Synthetics`? It's not a class.
I thought it would be some kind of namespace. I really don't have any opinion on the casing of this.

> IMO, we should simplify this, having an `index.ts` in each command directory that will handle what's getting exported to the lib (and rename the current index.ts as cli.ts for the automatic cli registration).

How would you change the current design from the existing? `index.ts` is currently being used by the commands: https://github.com/DataDog/datadog-ci/blob/master/src/index.ts#L22. I'm happy to turn all the `index.ts` into `cli.ts` but indeed, we might want to give this some thoughts.

An alternative would be to extract all the synthetics part into another package, which would be a lib and not a CLI, and import this package in `datadog-ci` which is only a CLI and not a lib.

Reply:
Agreed it need further planning and I think the best way to do so, both laying all solutions + discussing them would be a RFC. (not necessarily a long one, but I think it's the best way to communicate through this)
Let's create one and discuss it with all owners/maintainers of commands.